.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 .DS_Store
 dist/
+node_modules/
+JavaScript2/
```

Comment:
Why ignoring this? Why not delete it entirely?

Reply:
true, removed.